### PR TITLE
Install ember-getowner-polyfill and replace this.container with getOwner(this)

### PR DESCRIPTION
**Pull request by Alinaki**
Just in case of new Ember 2.3.0 Owner API.

### CODE REVIEW

File: addon/mixins/in-viewport.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ import canUseDOM from 'ember-in-viewport/utils/can-use-dom';
 import canUseRAF from 'ember-in-viewport/utils/can-use-raf';
 import isInViewport from 'ember-in-viewport/utils/is-in-viewport';
 import checkScrollDirection from 'ember-in-viewport/utils/check-scroll-direction';
+import getOwner from 'ember-getowner-polyfill';
+
```
Comment (rwjblue) [inline addon/mixins/in-viewport.js]: This extra line break is causing a JSCS failure.

**Comment by rwjblue** [comment]
Looks like one of the acceptance tests (and a JSCS test) is failing. Can you look into that?

**Comment by Alinaki** [comment]
I'll try

**Comment by poteto** [comment]
@homu r+

**Pull request merged**